### PR TITLE
Add basic readme

### LINKED ISSUE #302: Write up a decent README
Things that should be covered:
 * Link to matrix channel
 * Some screenshots showing different views
 * Deployment guide

### PR DESCRIPTION
**Pull request by robbevp**
Add basic readme.
Fix #302 

### Topics to cover
* [x] Info about app (quick first draft - same as in API)
* [ ] Screenshots for a few views
* [x] Deployment guide
* [x] Info for local development
* [x] Info for questions/bugs
* [x] Known issues

### CODE REVIEW

**Comment by chvp** [comment]
@robbevp do you think you'll get around to adding screenshots soon? Once the currently open pull requests on all the projects are merged I'd like to do 1.0 releases on all the projects. I feel like the project is ready for the 1.0 label. (Or do you still see any glaring problems preventing a 1.0 release?)

I can look into it as well if you don't foresee having time to do this in the (semi-)near future.

**Comment by robbevp** [comment]
> @robbevp do you think you'll get around to adding screenshots soon? Once the currently open pull requests on all the projects are merged I'd like to do 1.0 releases on all the projects. I feel like the project is ready for the 1.0 label. (Or do you still see any glaring problems preventing a 1.0 release?)
> 
> I can look into it as well if you don't foresee having time to do this in the (semi-)near future.

I'm not sure if I will have time to setup an environment to make screenshots with good artist images/album images. I'm still doubting whether we should go for one or two small artists/labels (but ask permission before) or go for really big artists.

Regarding 1.0 - I can't think of anything that is still missing.
Since the runner per location have a breaking change that needs api, web and api-client-js to match, it would be good to do merge those and then release 1.0

**Comment by chvp** [comment]
> Regarding 1.0 - I can't think of anything that is still missing. Since the runner per location have a breaking change that needs api, web and api-client-js to match, it would be good to do merge those and then release 1.0

Yes, I was definitely going to wait for those.

File: README.md [hunk 1]
```diff
@@ -1,29 +1,85 @@
...
+discussions](https://github.com/accentor/web/discussions) or in the
+[Matrix channel](https://matrix.to/#/!PCYHOaWItkVRNacTSv:vanpetegem.me?via=vanpetegem.me&via=matrix.org).
+
+Think you have noticed a bug or have a great idea for a feature?
+[Create an issue](https://github.com/accentor/web/issues/new/choose).
+
+## Known issues
+
+### Playback support depends on browser
+Playback of audio files depends on browser support entirerly. This can
```
Comment (chvp) [inline README.md:66]:
```suggestion
Playback of audio files depends on browser support entirely. This can
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,85 @@
...
+
+Once you have the API running, you have two options for the frontend.
+
+### Use a hosting service for static sites
+
+The project can be deployed to a static site hoster like Netlify.
+You only need to configure the following three things:
+* Build command: `yarn build`.
+* Output directory: `dist/`.
+* An environment variable for `VUE_APP_API_URL` pointing to the domain of your api.
```
Comment (chvp) [inline README.md:33]:
```suggestion
* An environment variable for `VUE_APP_API_URL` pointing to the domain of your api.
* Build command: `yarn build`.
* Output directory: `dist/`.
```

File: README.md [hunk 3]
```diff
@@ -1,29 +1,85 @@
...
+
+To use the web frontend, you already need to have an API running. You
+can read how to do that [here](https://github.com/accentor/api).
+
+Once you have the API running, you have two options for the frontend.
+
+### Use a hosting service for static sites
+
+The project can be deployed to a static site hoster like Netlify.
+You only need to configure the following three things:
```
Comment (chvp) [inline README.md:30]:
```suggestion
You only need the following three things:
```

**Review by chvp — APPROVED** [review]

**Pull request merged**